src/utils/helperFunctions.ts

```diff
@@ -120,3 +120,30 @@ export const removeColumns = <T extends object>(
     }
   });
 };
+
+export const isObject = (object: null) => {
+  return object != null && typeof object === 'object';
+};
+export const isObjectsEqual = (obja: object, objb: object) => {
```

Comment:
I see in the code `utils/json.ts` there is one function called `deepEqual`. Can you please check if it does the same comparison for object equality and use that one instead or if you think this one is better then swap that one with yours.

Reply:
They both perform the same task l want  equally but deepEqual covers more object types ,so l be using it instead of isObjectsEqual